enterprise/backend/src/metabase_enterprise/serialization/load.clj

```diff
@@ -443,6 +443,15 @@
                                                   (dissoc :dashboard_cards)
                                                   (assoc :collection_id (:collection context)
                                                          :creator_id    (default-user-id)))))
+        ;; MEGA HACK -- if `load` is ran with `--mode update` we should delete any Cards that were removed from a
+        ;; Dashboard (according to #20786). However there are literally zero facilities for doing this sort of thing in
+        ;; the current dump/load codebase. So for now we'll just delete ALL DashboardCards for the dumped Dashboard when
+        ;; running with `--mode update` and recreate them from the serialized definitions. This is definitely a wack way
+        ;; of doing things but no one actually understands how this code is supposed to work so this will have to do
+        ;; until we can come in here and clean things up. -- Cam 2022-03-24
+        _               (when (and (= (:mode context) :update)
```

Comment:
is there a `-` on this line?

Reply:
It's an underscore because it's inside of a `let`